infra/license/license-blocklist.json

```diff
@@ -0,0 +1,24 @@
+{
+    "Allowed": [
+        "0BSD",
+        "Apache-2.0",
+        "BSD-2-Clause",
+        "BSD-3-Clause",
+        "CC-BY-3.0",
+        "CC0-1.0",
+        "ISC",
+        "MIT",
+        "Unlicense",
+        "(MIT AND CC-BY-3.0)",
+        "(MIT OR CC0-1.0)"
+    ],
+
+    "Warning": [
+        "ODC-By-1.0",
+        "Python-2.0"
+    ],
+
+    "Denied": [
+        "GPL-3.0"
+    ]
+}
```

Comment:
I wonder whether there's any reference about the selection of this list. 
E.g., is there any guide from SE mentioning that, e.g., `0BSD` or `Unlicense` is allowed?

Reply:
I wrote my comment at internal secret repo :)